features/step_definitions/show_stats_steps.rb

```diff
@@ -59,7 +59,7 @@
     step "I run `#{command}`"
 
     result = 0
-    in_current_dir do
+    cd('.') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.